rcl/int32_multinode/main.c

```diff
@@ -0,0 +1,118 @@
+#include <rcl/rcl.h>
```

Comment:
The coding style is not consistent, please use the same style in `for` loop, `if` case, and so on.

Reply:
Done